ext4/tar2ext4/tar2ext4.go

```diff
@@ -195,11 +222,404 @@ func Convert(r io.Reader, w io.ReadWriteSeeker, options ...Option) error {
 	}
 
 	if p.appendVhdFooter {
-		return ConvertToVhd(w)
+		return ConvertToVhd(w, p.alignVHDToMB)
 	}
 	return nil
 }
 
+func findNextUnusedLogicalBlock(bytePosition uint64) uint64 {
+	block := uint64(bytePosition / compactext4.BlockSizeLogical)
+	if bytePosition%compactext4.BlockSizeLogical == 0 {
+		return block
+	}
+
+	return block + 1
+}
+
+func ConvertMultiple(multipleReaders []io.Reader, w io.ReadWriteSeeker, partitionGUIDs []string, diskGUIDString string, options ...Option) error {
```

Comment:
I'm not sure the `tar2ext4` package is the best place for this functionality. It would be nice to keep this package focused on the tar -> ext4 conversion, and not be aware of the fact that the file it's writing the ext4 fs to happens to have a GPT partitioning scheme. Maybe this code should live in the tool that is specifically used to produce a VHD from a set of tars?

Reply:
Sure, I'll look into moving this into the gpt package and then making a new tool :)